test/unit/Cardano/WalletSpec.hs

```diff
@@ -118,6 +124,21 @@ walletGetWrongIdProp (newWallet, corruptedWalletId) = monadicIO $ liftIO $ do
     attempt <- runExceptT $ readWallet wl corruptedWalletId
     attempt `shouldSatisfy` isLeft
 
+walletIdDeterministic
+    :: NewWallet
+    -> Property
+walletIdDeterministic newWallet = monadicIO $ liftIO $ do
+    (WalletLayerFixture _ _ widsA) <- liftIO $ setupFixture newWallet
+    (WalletLayerFixture _ _ widsB) <- liftIO $ setupFixture newWallet
+    widsA `shouldBe` widsB
+
+walletIdInjective
+    :: (NewWallet, NewWallet)
+    -> Property
+walletIdInjective (walletA, walletB) = monadicIO $ liftIO $ do
```

Comment:
it would persuade me more here if it would be imprinted here explicitly that a  substantial number of conflicts is tested (maybe adding cond for this is a good idea)

Reply:
What do you mean ? Verifying that the mnemonic are indeed different?

Reply:
To some extent, yes. We do not test the injectivity of `generateKeyFromSeed` and just take it as "granted" (since it's coming from `cardano-crypto` and it's tested there). 
One thing we could add however is to check that, mnemonics are actually different (it should be 100% coverage with QC `cover` so that we are sure that our generator is good enough).

Reply:
**to sum up**: I was just trying to say that it is informative for test to convey the message :  when we have some param(s) and they are different being n-tuple, then we have unique walletId;  when they are the same then the same walletId is expected. Some params does not contribute here (like name, gap) and for the same n-tuple they can be different and the walletId is the same. With that embedded in the test I do not have to look at the implementation to understand what are the rules behind walletID computation. And my next question was also continuing this investigation - are we sure `seed`, `secondFactor`, and `passprase` constitute this n-tuple?